spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -259,12 +259,10 @@ def update_app_details!(app_id, data)
     #   This can't be longer than 255 characters.
     # @param primary_language (String): If localized app information isn't available in an
     #   App Store territory, the information from your primary language will be used instead.
-    # @param version (String): The version number is shown on the App Store and should
-    #   match the one you used in Xcode.
     # @param sku (String): A unique ID for your app that is not visible on the App Store.
     # @param bundle_id (String): The bundle ID must match the one you used in Xcode. It
     #   can't be changed after you submit your first build.
-    def create_application!(name: nil, primary_language: nil, version: nil, sku: nil, bundle_id: nil, bundle_id_suffix: nil, company_name: nil, platform: nil)
+    def create_application!(name: nil, primary_language: nil, sku: nil, bundle_id: nil, bundle_id_suffix: nil, company_name: nil, platform: nil)
```

Comment:
This would be a breaking change when someone calls _spaceship_ scripts directly. Is there a good way in Ruby to deprecate a parameter?